src/layouts/generalLayout.tsx

```diff
@@ -4,14 +4,20 @@ import Navbar from '../components/nav';
 
 interface IProps {
   children: React.ReactNode;
+  title: string;
 }
 
-const GeneralLayout: React.FC<IProps> = ({ children }) => {
+const GeneralLayout: React.FC<IProps> = ({ children, title }) => {
   const className = classnames('general-layout');
+  const titleClass = classnames('general-title');
+  const rightContainer = classnames('general-container');
   return (
     <main className={className}>
```

Comment:
Since you just assign a constant value to it. I think you can just assign className here without define variables.
It may be like this:
```
<main className={classnames('general-layout')}>
```
Same recommendations for className assign below.

Reply:
fixed